demisto_sdk/commands/validate/tests/PB_validators_test.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+from demisto_sdk.commands.validate.validators.PB_validators.PB118_is_input_key_not_in_tasks import (
+    IsInputKeyNotInTasksValidator,
+)
+from demisto_sdk.commands.validate.tests.test_tools import create_playbook_object
+
+
+playbook = create_playbook_object()
+
+
+@pytest.mark.parametrize(
+    "input_in_task,input_in_section, expected_result",
+    [
+        ({"input1", "inputs3"}, {"input1"}, []),
+        (
+            {"input1", "inputs3"},
+            {"input1", "input2"},
+            "The playbook 'Phishing Investigation - Generic' contains the following inputs that are not used in any of its tasks: input2",
+        ),
+    ],
+)
+def test_is_valid_all_inputs_in_use(
+    mocker, input_in_task, input_in_section, expected_result
+):
+    """
+        Given:
+        - A playbook with inputs in some tasks and inputs defined in the inputs section
+
+        When:
+        - Validating the playbook
+
+        Then:
+        - If the inputs defined in the inputs section are not all in the playbook, return a list of errors, else return an empty list
+    ,"""
+
+    mocker.patch(
+        "demisto_sdk.commands.validate.validators.PB_validators.PB118_is_input_key_not_in_tasks.collect_all_inputs_in_use",
+        return_value=input_in_task,
+    )
+    mocker.patch(
+        "demisto_sdk.commands.validate.validators.PB_validators.PB118_is_input_key_not_in_tasks.collect_all_inputs_from_inputs_section",
+        return_value=input_in_section,
+    )
```

Comment:
Let's not use this.. we should be able to run the command if we initialized the playbook object correctly.

Reply:
If I understand what you mean, Done.